docs/deploy.md

```diff
@@ -236,3 +237,285 @@ The target mainchain address of a backward transfer has to be a valid address: s
 
 ```
 
+## Deploying a Contract using Hardhat
+To deploy a contract, you will use a deployment script that is specific for your smart contract. 
+Hardhat script.  Go to the `script/` folder and find the following deploy.ts file:
```

Comment:
The way `Hardhat script` is placed in here, I'm not too sure about. Is there supposed to be a period before this?